src/cmds/scripts/pbs_habitat.in

```diff
@@ -666,6 +675,20 @@ upgrade_pbs_schema_from_v1_2_0() {
 	fi
 }
 
+upgrade_pbs_schema_from_v1_3_0() {
+
+	${PGSQL_DIR}/bin/psql -p ${PBS_DATA_SERVICE_PORT} -d pbs_datastore -U ${PBS_DATA_SERVICE_USER} <<-EOF > /dev/null
+		DROP TABLE pbs.subjob_track
```

Comment:
Don't we need to back-up the data present in the table pbs.subjob_track before dropping(This data then can be used to populate pbs.job table by creating a row for each valid subjob into job table itself) it so that PBS upgrades will also retain subjobs ?

Reply:
Not required. Since for db upgrade the server will any how restart and as per our previous behavior we do not support subjob surviving server restart.